beps/bep_0007.rst

```diff
@@ -23,32 +24,21 @@ be interpreted as described in IETF `RFC 2119`_.
 
 .. _`RFC 2119`: http://tools.ietf.org/html/rfc2119
 
-Announce Parameter
-==================
-
-The client MAY add an ``&ipv6=`` parameter to the HTTP GET request it sends
-to the tracker. The value is either an IPv6 endpoint (address and port) or
-just an IPv6 address. In the case where only an address is supplied, the IPv6
-port is assumed to be the same as specified by the ``&port=`` parameter.
-
-The tracker SHOULD perform a NAT check on the IPv6 endpoint.
-
-In case the client contacts the tracker on an IPv6 interface, it may add
-an ``&ipv4=`` parameter with its IPv4 address or endpoint. The value MUST
-be either an IPv4 endpoint (address and port) or just an IPv4 address. If only
-an address is supplied, the port is assumed to be the same as the ``&port=``
-parameter.
+Announcing
+==========
 
-The endpoints are encoded as strings as defined by `RFC 2732`_.
+For each IP address the client can receive incoming peer connections over, it
```

Comment:
Perhaps *can* is the wrong word here. With ipv6 privacy extensions you could have many addresses. *intends to publish* might be better.